GMnetENGINE.gmx/scripts/htme_isLostConnection.gml

```diff
@@ -0,0 +1,28 @@
+///htme_isLostConnection()
+
+/*
+**  Description:
+**      This checks if a client or Server is Lost Connection
+**      This is very handy when you handle client exit if the server
+**      kicked you. Just check if this returns true then go to another room.
+**  
+**  Usage:
+**      <see above>
+**
+**  Arguments:
+**      <none>
+**
+**  Returns:
+**      true, if lost, false if not.
+**
+*/
+if instance_exists(global.htme_object) {
```

Comment:
Simple script to handle client disconnects. Just check if true then go to another room and the engine will be clean and ready for rumble again.